controllers/recordsController.js

```diff
@@ -0,0 +1,74 @@
+import AppError from "../errors/AppError";
+import {
+  save,
+  getRecordById,
+  updateById,
+  deleteRecordById,
+  getAllRecords
+} from "../models/recordsModel";
+
+const getAllRecord = async (req, res) => {
+  try {
+    const record = await getAllRecords(req.body.recordId);
+    res.status(200).send({ payload: record });
+  } catch (error) {
+    next(new AppError(error.message));
+  }
+};
+
+const getRecordsById = async (req, res, next) => {
+  try {
+    const record = await getRecordById(req.params.recordId);
+    if (record) {
+      res.status(200).send({ payload: record });
+    } else {
+      throw new AppError("Invalid Id");
+    }
+  } catch (error) {
+    next(new AppError(error.message));
+  }
+};
+
+const addRecords = async (req, res, next) => {
+  try {
+    const record = await save({
+      questionId: req.body.questionId,
+      answers: req.body.answersId
+    });
+    res.status(201).send({ message: { payload: record } });
```

Comment:
better send payload with message that records was created and record that was added to DB